BrainPortal/app/models/message.rb

```diff
@@ -329,4 +333,26 @@ def self.parse_markup(string) #:nodoc:
     arr.join.html_safe
   end
 
+  def track_resource_usage_update
+    if self.message_type == "communication"
```

Comment:
Keep code structure simpler. Unindent like this:

```ruby
def method
   return true if condition_where_nothing_happens
   stuff
   stuff
end
```

Reply:
changed. Not sure is it simple (cyclometric complexity, more difficult to add new branches, the least interesting case on top of the method description) yet shorter